test/specs/collections/Table/TableCell-test.js

```diff
@@ -42,5 +42,11 @@ describe('TableCell', () => {
 
       wrapper.text().should.equal(content)
     })
+
+    it('render 0 value', () => {
```

Comment:
It makes sense to move it to [commonTests.rendersChildren()](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/test/specs/commonTests.js#L454). It will test all components that renders children.